presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java

```diff
@@ -920,19 +947,52 @@ public void testInsertIntoPartitionedTable()
 
     @Test
     public void testCreateBucketTableAsSelect()
+    {
+        Session session = buildSessionForTableWrite();
+        // Generate temporary table name for bucketed table.
+        String bucketedOrdersTableName = generateRandomTableName();
+        // Clean up if temporary table already exists.
+        dropTableIfExists(bucketedOrdersTableName);
+
+        String[] tableFormats = {"DWRF", "PARQUET"};
```

Comment:
This is repeated in multiple places. Maybe make it a constant member variable in the class?